app/Models/Foundation/Summit/Summit.php

```diff
@@ -65,6 +65,22 @@ class Summit extends SilverstripeBaseModel
 
     use GetDefaultValueFromConfig;
 
+    public static $default_report_setting = [
+        'scan_date',
+        'attendee_first_name',
+        'attendee_last_name',
+        'attendee_email',
+        'attendee_company',
+        'attendee_extra_questions' => [
+            'id',
+            'name'
+        ],
+        'extra_questions' => [
```

Comment:
this is not correct the same as above, as the default is all, we could create a will card like 
extra_questions => ['*'] to indicate all as default